.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+
+## Fixes Issue
+
+Read [Contributing to Sparkeats](https://github.com/sparkbox/sparkeats/blob/main/CONTRIBUTING.md).
+
+This pull request addresses issue #[ISSUE_NUMBER].
+
+# What changed
+
+<!-- Describe the changes in the pull request. -->
+<!-- List other components or functionality impacted by the changes. -->
+
+## How to validate
+<!-- List the steps to validate whether the changes meet the issue's requirements or not. -->
+
+## Screenshots
```

Comment:
I think this section can also be removed and you can move the comment to the `What changed` section instead because screenshots or videos can be used to show what changed.

Reply:
maybe will delete the how-to validate section but not the screenshots

Reply:
I have done some more research and reviewed some of the pull request templates from the [top open source repositories](https://github.com/EvanLi/Github-Ranking/blob/master/Top100/Top-100-stars.md).

This research has reinforced my preference for a minimal pull request template. I do not want to add more tasks or sections to the pull request template than are necessary, especially since Sparkeats is a small project.

The [React pull request template](https://github.com/facebook/react/blob/main/.github/PULL_REQUEST_TEMPLATE.md
) is a good example of what I have in mind.

It has the same two sections that I am proposing: "Summary" which is the same as "What changed" and "How did you test the change?" which is the same as "How to validate". Other information such as including screenshots is a visual example of "What changed" and can be included in a comment as in [my example above](https://github.com/sparkbox/sparkeats/pull/397/files#r990877737). I do think that "How did you test the change?" is more useful than "How to validate", however, so I do think we should change that.

What do you think about these two sections:

1. `What changed`
2. `How did you test the change?`

Also, thank you taking the time to discuss this with me @neutr0n420. If you prefer to use other sections, would you mind sharing your reasoning for them so that I could better understand your perspective?

Reply:
- This minimum setup looks good to me, but I think we should also incorporate the screenshot or video section for more clarity. 
- It does look good to me to change "How to validate" to the "What changed".
- I am worried about the question "How did you test the changes". I personally wasn't able to understand the question can you please give some brief about it?
And thank you for taking this much time, and discussing it with me and also thank you for even considering me for this issue.

Reply:
Alright, I see how it will be more explicit with a `Screenshots` section. Let's leave it in then.

I meant to use "How did you test the changes" instead of "How to validate".  The purpose is to tell the PR reviewer how you tested the changes so that they can also test to make sure it works. I got this idea from the React pull request template.

So how about these three sections @neutr0n420?

```markdown
`What changed?`

`How did you test the changes?`

`Screenshots`
```

Reply:
Really looking neat to me as well. Modifying the changes in the file and pushing it.